### ISSUE TITLE
Shea Manga returning "Trust anchor for certification path not found" after #10951

### ISSUE THREAD

**Issue by MoonLGH** [tachiyomiorg/tachiyomi-extensions#11475]
### Source information

Shea  Manga 1.2.17

### Steps to reproduce

1. go to Shea Manga extensions 

### Expected behavior

return lists of manga

### Actual behavior

return "Trust anchor for certification path not found"

### Tachiyomi version

0.12.3.6

### Android version

Android 11 - OneUi 3.1

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by MoonLGH** [comment]
I believe the problem comes because
        `.dns(Dns.SYSTEM)`
because I experience the same thing while making another extension

**Comment by MoonLGH** [comment]
edit : its because the didnt have any ssl cert

**Comment by alessandrojean** [comment]
Closing since it's something the website needs to fix.
